client/payment-details/summary/test/__snapshots__/index.tsx.snap

```diff
@@ -272,6 +310,22 @@ exports[`PaymentDetailsSummary renders a charge with subscriptions 1`] = `
         <div
           class="payment-details-summary__section"
         >
+          <div
```

Comment:
Maybe this is why we see changes in the status of the payment?

Reply:
During the time of this PR there was indeed another one added related to Fraud review. The snapshot related to that seems to have gotten inserted. I hope I am reading this correctly?